app/Api/page.tsx

```diff
@@ -0,0 +1,127 @@
+'use client'
+import Image from "next/image"
+import useState  from "react-usestateref";
```

Comment:
This useState() usage is deprecated (old and no longer used). Make some changes to use the new, updated hooks:
```
import { useState } from "react";

...some code

const[messages, setMessages] = useState<MessageProps[]>([]);
setMessages([...messages, myMessage]);

...some other code
```